walmart/walmart.py

```diff
@@ -300,6 +300,22 @@ class Orders(Resource):
 
     path = 'orders'
 
+    def all(self, **kwargs):
+        try:
+            return super(Orders, self).all(**kwargs)
+        except requests.exceptions.HTTPError as exc:
+            if exc.response.status_code == 404:
+                # If no new orders are there on walmart, it throws 404. In
```

Comment:
new orders or any orders that match the filters?

Reply:
sorry any orders.
Updated comment as well as commit message